fgd/brush/trigger/trigger_momentum_timer_cancel.fgd

```diff
@@ -0,0 +1,6 @@
+@SolidClass appliesto(MOMENTUM)
+= trigger_momentum_timer_cancel : "Zone that cancels the timer."
+	[
+	filtername(string) : "Filter Name" : : "filter_activator entity to use for this zone."
+	track_number(integer) : "Track Number" : 0 : "Track number that this entity belongs to. 0 = main map, 1+ = bonus zone (of that number)."
```

Comment:
Maybe (nit) but the word "zone" here could be misinterpreted as "the number of the zone within the track". 
```suggestion
	track_number(integer) : "Track Number" : 0 : "Track number that this entity belongs to. 0 = main map, 1+ = bonus (of that number)."
```
Alternatively it could be replaced with the word "track"

(also applies to the same field on other zones)